src/foraging_gui/Foraging.py

```diff
@@ -1000,6 +1051,8 @@ def _GetSettings(self):
                 4:'D'
             }
             self.current_box='{}-{}'.format(self.current_box,mapper[self.box_number])
+        self.Other_current_box=self.current_box
```

Comment:
What is `self.Other_current_box`?

Reply:
Just save the `current_box` to behavior json file. The code will automatically save field with prefix 'Other_'